cmd/bridge/main.go

```diff
@@ -560,6 +572,21 @@ func main() {
 		bridge.FlagFatalf("k8s-mode", "must be one of: service-account, bearer-token, oidc, openshift")
 	}
 
+	srv.PluginsHandler = plugins.NewPluginsHandler(
+		srv.ServiceAccountToken,
+		&url.URL{
+			Scheme: k8sEndpoint.Scheme,
+			Host:   k8sEndpoint.Host,
+			Path:   "/apis/console.openshift.io/v1/consoleplugins",
+		},
+		&http.Client{
```

Comment:
is this client supposed to have the same `TLSClientConfig` as the kube client?

Reply:
it should have `serviceProxyTLSConfig`, which is set based on the `off-cluster` or `in-cluster` mode in which the console server is running